src/parser/query_result.c

```diff
@@ -836,14 +836,31 @@ pt_get_node_title (PARSER_CONTEXT * parser, const PT_NODE * col, const PT_NODE *
 			  if (original_name)
 			    {
 			      /* strip off classname.* */
-			      name += strlen (node->info.name.original) + 1;
+			      if (strchr (node->info.name.original, '.'))
+				{
+				  name = strchr (original_name, '.');
```

Comment:
Are node->info.name.original and original_name the same value?

Reply:
Yes, it is not changed after setting as below

```
728: original_name = name = pt_print_alias (parser, col);
800: node = node->info.dot.arg1;
```